src/main/scala/com/gu/CheckPriceRisePreConditions.scala

```diff
@@ -0,0 +1,54 @@
+package com.gu
+
+import com.gu.FileImporter.PriceRise
+
+trait PriceRisePreCondition
+case object SubscriptionIsAutoRenewable extends PriceRisePreCondition
+case object SubscriptionIsActive extends PriceRisePreCondition
+case object DeliveryRegionMatchesCurrency extends PriceRisePreCondition
+case object PriceRiseDateIsOnInvoicedPeriodEndDate extends PriceRisePreCondition
+case object ImportHasCorrectCurrentPrice extends PriceRisePreCondition
+case object TargetPriceRiseIsNotMoreThanTheCap extends PriceRisePreCondition
+case object TargetPriceRiseIsNotMoreThanDefaultProductRatePlanChargePrice extends PriceRisePreCondition
+case object TargetPriceRiseIsMoreThanTheCurrentPrice extends PriceRisePreCondition
+case object ThereDoesNotExistAFutureAmendmentOnThePriceRiseDate extends PriceRisePreCondition
```

Comment:
This might need a bit of thought. I think it only applies in some conditions, and others we might want to let through.

Reply:
I would be in favour of skipping over these and picking them up in a second pass - I expect it will be a fairly unusual case?